### PR TITLE
feat(French-Starr): add VOQ (Version Originale Québecoise | Quebec Original Version) CF

### LINKED ISSUE #1572: feat(French-Starr): add `VOQ` (Version Originale Québecoise | Quebec Original Version) CF
There's missing a custom format for VOQ

Origin user: gravelfreeman
Origin message: [Discord Server Link](https://discord.com/channels/492590071455940612/1033347272647385169/1155965661021995150)
Reporting user: nicetoseeyou
Reporting message: [Discord Server Link](https://discord.com/channels/492590071455940612/1033347272647385169/1155967347480350730)

### PR DESCRIPTION
**Pull request by NiceTSY**
# Pull Request

## Purpose

Add VOQ CF for Original Quebec movie as informative CF
Resolves #1572 

## Approach

- [x] Added the CF for Radarr
- [x] Updated Radarr's Collection of Format
- [x] Updated French Radarr's Information
- [x] Added the CF for Sonarr
- [x] Updated Sonarr's Collection of Format
- [x] Updated French Sonarr's Information

## Requirements

- [x] These changes meet the standards for [contributing](https://github.com/TRaSH-Guides/Guides/blob/master/CONTRIBUTING.md).
- [x] I have read the [code of conduct](https://github.com/TRaSH-Guides/Guides/blob/master/.github/CODE_OF_CONDUCT.md).

### CODE REVIEW

File: docs/json/radarr/cf/french-voq.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "trash_id": "f7caa1942be5cc547c266bd3dbc2cda9",
```
Comment (nuxencs) [inline docs/json/radarr/cf/french-voq.json:2]: are there supposed to be no scores? if so, looks good to me 😄

File: docs/json/sonarr/cf/french-voq.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
+  "trash_id": "802dd70b856c423a9b0cb7f34ac42be1",
```
Comment (nuxencs) [inline docs/json/sonarr/cf/french-voq.json:2]: are there supposed to be no scores? if so, looks good to me 😄

File: docs/json/radarr/cf/french-voq.json (hunk 1 above)
Comment (NiceTSY) [reply docs/json/radarr/cf/french-voq.json:2]: Yes, both are informative. It is like all the other French audio CFs. They exist just for people to know which French they will potentially get 😅

File: docs/json/sonarr/cf/french-voq.json (hunk 2 above)
Comment (NiceTSY) [reply docs/json/sonarr/cf/french-voq.json:2]: Yes, both are informative. It is like all the other French audio CFs. They exist just for people to know which French they will potentially get 😅

**Review by nuxencs — APPROVED** [review]

**Pull request merged**